app/frontend/src/components/designer/FormSettings.vue

```diff
@@ -459,8 +459,11 @@
             <template v-slot:no-data>
               <v-list-item>
                 <v-list-item-content>
-                  <v-list-item-title :lang="lang">
-                    {{ $t('trans.formSettings.pressToAddMultiEmail') }}
+                  <v-list-item-title>
+                    <span
+                      v-html="$t('trans.formSettings.pressToAddMultiEmail')"
+                      :lang="lang"
+                    />
```

Comment:
Bug fix - the HTML in pressToAddMultiEmail is being escaped:
![image](https://github.com/bcgov/common-hosted-form-service/assets/35532993/62911ea2-7905-4516-b260-e58fc6e0da1a)